addons/website_slides/static/src/xml/activity.xml

```diff
@@ -19,4 +19,32 @@
             </t>
         </t>
     </t>
+
+    <t t-name="slides.Slide" owl="1">
+        <div t-if="slide" class="o_Slide">
+            <button
+                role="button"
+                class="o_Activity_toolButton o_Activity_GrantAccessButton btn btn-link"
+                t-on-click="_onGrantAccess"
+            >
+                <i class="fa fa-check"/> Grant Access
+            </button>
+            <button
+                role="button"
+                class="o_Activity_toolButton o_Activity_RefuseAccessButton btn btn-link"
+                t-on-click="_onRefuseAccess"
+            >
+                <i class="fa fa-times"/> Refuse Access
+            </button>
+        </div>
+    </t>
+
+    <t t-inherit="mail.Activity" t-inherit-mode="extension">
+        <xpath expr="//*[@name='tools']" position="replace">
+            <t t-if="activity.slide">
+                <Slide slideLocalId="activity.slide.localId"/>
```

Comment:
This component name `Slide` doesn't make sense actually, it should be named `SlideAccessRequest` or something similar.

Probably same remark for the model, and that might explain why you did `one2one` with activity, and then it was correct if "slide" is actually "slide access request".